spec/Functions.tex

```diff
@@ -1076,9 +1085,9 @@ \subsection{Determining Visible Functions}
 function} if the name of the function is the same as the name of the
 function call and the function is defined in the same scope as the
 function call or a lexical outer scope of the function call, or if the
-function is defined in a module that is used from the same scope as
-the function call or a lexical outer scope of the function call.
-Function visibility in generic functions is discussed
+function is publicly defined in a module that is used from the same
```

Comment:
I suggest "function has public visibility and is defined in a module ..."

Rationale: we do not have a definition for "publicly defined".